tests/behaviour/expectations/behaviour.ts

```diff
@@ -187,6 +187,17 @@ export const expectations = flatten(
             'C',
           ),
         ]),
+        new Dir('using_for', [
+          File.Simple('simple', [
+            Expect.Simple('bar', [], ['6', '0']),
+            Expect.Simple('fum', [], ['60', '0']),
```

Comment:
These tests could do with some more descriptive test names. For example `fum` could be named something lke `callOnConversion` and `bar` could be `callOnIdentifier`, something along those lines.